tests/test_axis_limits.py

```diff
@@ -1,101 +1,334 @@
+            "full_with_axis_dependencies"
+        ]["data"]
+        if not seeds.isdisjoint(deps):
+            out.add(key)
+    return out
+
+
+not_implemented_limits = grow_seeds(
+    not_implemented_limits,
+    data_index["desc.equilibrium.equilibrium.Equilibrium"].keys() - not_finite_limits,
+)
+
+
+def _skip_this(eq, name):
+    return (
+        name in not_implemented_limits
+        or (eq.atomic_number is None and "Zeff" in name)
+        or (eq.electron_temperature is None and "Te" in name)
+        or (eq.electron_density is None and "ne" in name)
+        or (eq.ion_temperature is None and "Ti" in name)
+        or (eq.pressure is not None and "<J*B> Redl" in name)
+        or (eq.current is None and ("iota_num" in name or "iota_den" in name))
+    )
+
+
+def assert_is_continuous(
+    eq,
+    names,
+    delta=5e-5,
+    rtol=1e-4,
+    atol=1e-6,
+    desired_at_axis=None,
+    kwargs=None,
+):
+    """
+    Asserts that the rho=0 axis limits of names are continuous extensions.
+
+    Parameters
+    ----------
+    eq : Equilibrium
+        The equilibrium object used for the computation.
+    names : list, str
+        A list of names of the quantities to test for continuity.
+    delta: float, optional
+        Max distance from magnetic axis.
+        Smaller values accumulate finite precision error and fitting issues.
+    rtol : float, optional
+        Relative tolerance.
+    atol : float, optional
+        Absolute tolerance.
+    desired_at_axis : float, optional
+        If not provided, the values are extrapolated with a polynomial fit.
+    kwargs : dict, optional
+        Keyword arguments to override the parameters above for specific names.
+        The dictionary should have the following structure:
+        {
+            "name1": {
+                "rtol": custom_rtol1,
+                "atol": custom_atol1,
+                "desired_at_axis": custom_desired_at_axis1
+            },
+            "name2": {"rtol": custom_rtol2},
+            ...
+        }
+
+    """
+    if kwargs is None:
+        kwargs = {}
+    if isinstance(names, str):
+        names = [names]
+    # TODO: remove when boozer transform works with multiple surfaces
+    names = [x for x in names if not ("Boozer" in x or "_mn" in x or x == "B modes")]
+
+    num_points = 15
+    rho = np.linspace(start=0, stop=1, num=num_points) * delta
+    grid = LinearGrid(rho=rho, M=5, N=5, NFP=eq.NFP, sym=eq.sym)
+    assert grid.axis.size
+    integrate = surface_integrals_map(grid)
+    data = eq.compute(names, grid=grid)
+
+    data_index_eq = data_index["desc.equilibrium.equilibrium.Equilibrium"]
+    for name in names:
+        if _skip_this(eq, name) or data_index_eq[name]["coordinates"] == "":
+            continue
+        # make single variable function of rho
+        if data_index_eq[name]["coordinates"] == "r":
+            profile = data[name]
+        else:
+            profile = np.where(
+                np.isnan(data[name]),
```

Comment:
Technically the correct way to do this is `integrate(np.isnan(data[name])).astype(bool)` instead of `np.isnan(data[name])`.